INFO: 2020-01-12 13:34:51,250: main() 'Using profile folder /home/harold/.config/variety/'
INFO: 2020-01-12 13:34:51,278: perform_upgrade() 'Last run version was 0.8.2 or earlier, current version is 0.8.2'
INFO: 2020-01-12 13:34:51,724: process_command() 'Received command: ['--profile', '/home/harold/.config/variety/']'
INFO: 2020-01-12 13:34:51,745: load_banned() 'Missing or invalid banned URLs list, no URLs will be banned'
INFO: 2020-01-12 13:34:51,795: start() 'Using data_path /usr/share/variety'
INFO: 2020-01-12 13:34:51,795: load() 'Jumble loading'
INFO: 2020-01-12 13:34:51,796: _walk_modules() 'Jumble loading module in __init__ from /usr/lib/python3.8/site-packages/variety/plugins/builtin/__init__.py'
INFO: 2020-01-12 13:34:51,798: _walk_modules() 'Jumble loading module in GoodreadsSource from /usr/lib/python3.8/site-packages/variety/plugins/builtin/quotes/GoodreadsSource.py'
INFO: 2020-01-12 13:34:51,833: load() 'Jumble found plugin class: <class 'GoodreadsSource.GoodreadsSource'>: {'name': 'Goodreads', 'description': '从 Goodreads.com 获取箴言', 'author': 'Peter Levi', 'version': '0.1'}'
INFO: 2020-01-12 13:34:51,837: _walk_modules() 'Jumble loading module in UrbanDictionarySource from /usr/lib/python3.8/site-packages/variety/plugins/builtin/quotes/UrbanDictionarySource.py'
INFO: 2020-01-12 13:34:51,845: load() 'Jumble found plugin class: <class 'UrbanDictionarySource.UrbanDictionarySource'>: {'name': 'Urban Dictionary', 'description': 'Displays definitions from Urban Dictionary', 'author': 'James Miller', 'version': '0.1'}'
INFO: 2020-01-12 13:34:51,846: _walk_modules() 'Jumble loading module in __init__ from /usr/lib/python3.8/site-packages/variety/plugins/builtin/quotes/__init__.py'
INFO: 2020-01-12 13:34:51,849: _walk_modules() 'Jumble loading module in QuotationsPageSource from /usr/lib/python3.8/site-packages/variety/plugins/builtin/quotes/QuotationsPageSource.py'
INFO: 2020-01-12 13:34:51,864: load() 'Jumble found plugin class: <class 'QuotationsPageSource.QuotationsPageSource'>: {'name': 'TheQuotationsPage.com', 'description': '从 QuotationsPage.com 获取箴言', 'author': 'Peter Levi', 'version': '0.1'}'
INFO: 2020-01-12 13:34:51,866: _walk_modules() 'Jumble loading module in LocalFilesSource from /usr/lib/python3.8/site-packages/variety/plugins/builtin/quotes/LocalFilesSource.py'
INFO: 2020-01-12 13:34:51,868: load() 'Jumble found plugin class: <class 'LocalFilesSource.LocalFilesSource'>: {'name': 'Local text files', 'description': 'Displays quotes, defined in local text files.\nPut your own txt files in: /home/harold/.config/variety/pluginconfig/quotes/.\nThe file format is:\n\nquote -- author\n.\nsecond quote -- another author\n.\netc...\n\nExample: http://rvelthuis.de/zips/quotes.txt', 'author': 'Peter Levi', 'version': '0.1'}'
INFO: 2020-01-12 13:34:51,874: _walk_modules() 'Jumble loading module in FortuneSource from /usr/lib/python3.8/site-packages/variety/plugins/builtin/quotes/FortuneSource.py'
INFO: 2020-01-12 13:34:51,876: load() 'Jumble found plugin class: <class 'FortuneSource.FortuneSource'>: {'name': 'UNIX fortune program', 'description': 'Displays quotes using the UNIX fortune program. You may want to install additional fortune packs, e.g. fortunes-bofh-excuses.', 'author': 'Dan Jones', 'version': '0.1'}'
INFO: 2020-01-12 13:34:51,882: _walk_modules() 'Jumble loading module in DesktopprDownloader from /usr/lib/python3.8/site-packages/variety/plugins/builtin/downloaders/DesktopprDownloader.py'
INFO: 2020-01-12 13:34:51,888: load() 'Jumble found plugin class: <class 'DesktopprDownloader.DesktopprDownloader'>: {'name': 'DesktopprDownloader', 'description': 'Random wallpapers from Desktoppr.co', 'author': 'Peter Levi', 'version': '0.1'}'
INFO: 2020-01-12 13:34:51,891: _walk_modules() 'Jumble loading module in MediaRSSSource from /usr/lib/python3.8/site-packages/variety/plugins/builtin/downloaders/MediaRSSSource.py'
INFO: 2020-01-12 13:34:51,901: load() 'Jumble found plugin class: <class 'MediaRSSSource.MediaRSSSource'>: {'name': 'MediaRSSSource', 'description': 'Configurable source for fetching images from MediaRSS feeds', 'author': 'Peter Levi', 'version': '0.1'}'
INFO: 2020-01-12 13:34:51,906: _walk_modules() 'Jumble loading module in APODDownloader from /usr/lib/python3.8/site-packages/variety/plugins/builtin/downloaders/APODDownloader.py'
INFO: 2020-01-12 13:34:51,909: load() 'Jumble found plugin class: <class 'APODDownloader.APODDownloader'>: {'name': 'APODDownloader', 'description': 'NASA Astro Pic of the Day', 'author': 'Peter Levi', 'version': '0.1'}'
INFO: 2020-01-12 13:34:51,909: _walk_modules() 'Jumble loading module in __init__ from /usr/lib/python3.8/site-packages/variety/plugins/builtin/downloaders/__init__.py'
INFO: 2020-01-12 13:34:51,910: _walk_modules() 'Jumble loading module in RedditSource from /usr/lib/python3.8/site-packages/variety/plugins/builtin/downloaders/RedditSource.py'
INFO: 2020-01-12 13:34:51,913: load() 'Jumble found plugin class: <class 'RedditSource.RedditSource'>: {'name': 'RedditSource', 'description': 'Configurable source for fetching images from Reddit', 'author': 'Peter Levi', 'version': '0.1'}'
INFO: 2020-01-12 13:34:51,914: _walk_modules() 'Jumble loading module in WallhavenDownloader from /usr/lib/python3.8/site-packages/variety/plugins/builtin/downloaders/WallhavenDownloader.py'
INFO: 2020-01-12 13:34:51,916: _walk_modules() 'Jumble loading module in WallhavenSource from /usr/lib/python3.8/site-packages/variety/plugins/builtin/downloaders/WallhavenSource.py'
INFO: 2020-01-12 13:34:51,918: load() 'Jumble found plugin class: <class 'WallhavenSource.WallhavenSource'>: {'name': 'MediaRSSSource', 'description': 'Configurable source for fetching images from Wallhaven.cc', 'author': 'Peter Levi', 'version': '0.1'}'
INFO: 2020-01-12 13:34:51,919: _walk_modules() 'Jumble loading module in ChromeOSWallpapersDownloader from /usr/lib/python3.8/site-packages/variety/plugins/builtin/downloaders/ChromeOSWallpapersDownloader.py'
INFO: 2020-01-12 13:34:51,920: load() 'Jumble found plugin class: <class 'ChromeOSWallpapersDownloader.ChromeOSWallpapersDownloader'>: {'name': 'ChromeOsWallpapersDownloader', 'description': 'Chrome OS Wallpapers', 'author': 'Peter Levi', 'version': '0.1'}'
INFO: 2020-01-12 13:34:51,921: _walk_modules() 'Jumble loading module in EarthDownloader from /usr/lib/python3.8/site-packages/variety/plugins/builtin/downloaders/EarthDownloader.py'
INFO: 2020-01-12 13:34:51,922: load() 'Jumble found plugin class: <class 'EarthDownloader.EarthDownloader'>: {'name': 'EarthDownloader', 'description': 'World Sunlight Map - live wallpaper from Die.net', 'author': 'Peter Levi', 'version': '0.1'}'
INFO: 2020-01-12 13:34:51,923: _walk_modules() 'Jumble loading module in UnsplashDownloader from /usr/lib/python3.8/site-packages/variety/plugins/builtin/downloaders/UnsplashDownloader.py'
INFO: 2020-01-12 13:34:51,925: load() 'Jumble found plugin class: <class 'UnsplashDownloader.UnsplashDownloader'>: {'name': 'UnsplashDownloader', 'description': 'High-resolution photos from Unsplash.com', 'author': 'Peter Levi', 'version': '0.1'}'
INFO: 2020-01-12 13:34:51,925: _walk_modules() 'Jumble loading module in EarthviewDownloader from /usr/lib/python3.8/site-packages/variety/plugins/builtin/downloaders/EarthviewDownloader.py'
INFO: 2020-01-12 13:34:51,927: load() 'Jumble found plugin class: <class 'EarthviewDownloader.EarthviewDownloader'>: {'name': 'EarthviewDownloader', 'description': 'Google Earth View Wallpapers', 'author': 'Peter Levi', 'version': '0.1'}'
INFO: 2020-01-12 13:34:51,927: _walk_modules() 'Jumble loading module in MediaRSSDownloader from /usr/lib/python3.8/site-packages/variety/plugins/builtin/downloaders/MediaRSSDownloader.py'
INFO: 2020-01-12 13:34:51,932: _walk_modules() 'Jumble loading module in UnsplashConfigurableSource from /usr/lib/python3.8/site-packages/variety/plugins/builtin/downloaders/UnsplashConfigurableSource.py'
INFO: 2020-01-12 13:34:51,935: load() 'Jumble found plugin class: <class 'UnsplashConfigurableSource.UnsplashConfigurableSource'>: {'name': 'UnsplashConfigurableSource', 'description': 'Configurable source for fetching photos from Unsplash.com', 'author': 'Peter Levi', 'version': '0.1'}'
INFO: 2020-01-12 13:34:51,936: _walk_modules() 'Jumble loading module in BingDownloader from /usr/lib/python3.8/site-packages/variety/plugins/builtin/downloaders/BingDownloader.py'
INFO: 2020-01-12 13:34:51,937: load() 'Jumble found plugin class: <class 'BingDownloader.BingDownloader'>: {'name': 'BingDownloader', 'description': 'Bing Photo of the Day', 'author': 'Peter Levi', 'version': '0.1'}'
INFO: 2020-01-12 13:34:51,938: _walk_modules() 'Jumble loading module in RedditDownloader from /usr/lib/python3.8/site-packages/variety/plugins/builtin/downloaders/RedditDownloader.py'
INFO: 2020-01-12 13:34:51,964: log_options() 'Loaded options:'
INFO: 2020-01-12 13:34:51,965: log_options() 'change_enabled = True'
INFO: 2020-01-12 13:34:51,965: log_options() 'change_interval = 300'
INFO: 2020-01-12 13:34:51,965: log_options() 'change_on_start = False'
INFO: 2020-01-12 13:34:51,965: log_options() 'clipboard_enabled = False'
INFO: 2020-01-12 13:34:51,965: log_options() 'clipboard_hosts = ['wallhaven.cc', 'ns223506.ovh.net', 'wallpapers.net', 'flickr.com', 'imgur.com', 'deviantart.com', 'interfacelift.com', 'vladstudio.com']'
INFO: 2020-01-12 13:34:51,965: log_options() 'clipboard_use_whitelist = True'
INFO: 2020-01-12 13:34:51,965: log_options() 'clock_date_font = Sans 32'
INFO: 2020-01-12 13:34:51,965: log_options() 'clock_enabled = True'
INFO: 2020-01-12 13:34:51,965: log_options() 'clock_filter = -density 100 -font `fc-match -f '%{file[0]}' '%CLOCK_FONT_NAME'` -pointsize %CLOCK_FONT_SIZE -gravity SouthEast -fill '#00000044' -annotate 0x0+[%HOFFSET+58]+[%VOFFSET+108] '%H:%M' -fill white -annotate 0x0+[%HOFFSET+60]+[%VOFFSET+110] '%H:%M' -font `fc-match -f '%{file[0]}' '%DATE_FONT_NAME'` -pointsize %DATE_FONT_SIZE -fill '#00000044' -annotate 0x0+[%HOFFSET+58]+[%VOFFSET+58] '%A, %B %d' -fill white -annotate 0x0+[%HOFFSET+60]+[%VOFFSET+60] '%A, %B %d''
INFO: 2020-01-12 13:34:51,965: log_options() 'clock_font = Sans Bold 70'
INFO: 2020-01-12 13:34:51,965: log_options() 'configfile = /home/harold/.config/variety/variety.conf'
INFO: 2020-01-12 13:34:51,965: log_options() 'copyto_enabled = False'
INFO: 2020-01-12 13:34:51,965: log_options() 'copyto_folder = Default'
INFO: 2020-01-12 13:34:51,966: log_options() 'desired_color = [160, 160, 160]'
INFO: 2020-01-12 13:34:51,966: log_options() 'desired_color_enabled = False'
INFO: 2020-01-12 13:34:51,966: log_options() 'download_folder = /home/harold/.config/variety/Downloaded'
INFO: 2020-01-12 13:34:51,966: log_options() 'download_preference_ratio = 0.9'
INFO: 2020-01-12 13:34:51,966: log_options() 'favorites_folder = /home/harold/.config/variety/Favorites'
INFO: 2020-01-12 13:34:51,966: log_options() 'favorites_operations = [['Downloaded', 'Copy'], ['Fetched', 'Move'], ['Others', 'Copy']]'
INFO: 2020-01-12 13:34:51,966: log_options() 'fetched_folder = /home/harold/.config/variety/Fetched'
INFO: 2020-01-12 13:34:51,966: log_options() 'filters = [[False, 'Keep original', ''], [False, 'Grayscale', '-type Grayscale'], [False, 'Heavy blur', '-blur 120x40'], [False, 'Oil painting', '-paint 6'], [False, 'Charcoal painting', '-charcoal 3'], [False, 'Pointilism', '-spread 10 -noise 3'], [False, 'Pixellate', '-scale 3% -scale 3333%']]'
INFO: 2020-01-12 13:34:51,966: log_options() 'icon = Light'
INFO: 2020-01-12 13:34:51,966: log_options() 'lightness_enabled = False'
INFO: 2020-01-12 13:34:51,966: log_options() 'lightness_mode = 0'
INFO: 2020-01-12 13:34:51,966: log_options() 'min_rating = 4'
INFO: 2020-01-12 13:34:51,966: log_options() 'min_rating_enabled = False'
INFO: 2020-01-12 13:34:51,966: log_options() 'min_size = 80'
INFO: 2020-01-12 13:34:51,966: log_options() 'min_size_enabled = False'
INFO: 2020-01-12 13:34:51,967: log_options() 'quota_enabled = True'
INFO: 2020-01-12 13:34:51,967: log_options() 'quota_size = 1000'
INFO: 2020-01-12 13:34:51,967: log_options() 'quotes_authors = '
INFO: 2020-01-12 13:34:51,967: log_options() 'quotes_bg_color = [80, 80, 80]'
INFO: 2020-01-12 13:34:51,967: log_options() 'quotes_bg_opacity = 55'
INFO: 2020-01-12 13:34:51,967: log_options() 'quotes_change_enabled = False'
INFO: 2020-01-12 13:34:51,967: log_options() 'quotes_change_interval = 300'
INFO: 2020-01-12 13:34:51,967: log_options() 'quotes_disabled_sources = ['']'
INFO: 2020-01-12 13:34:51,967: log_options() 'quotes_enabled = False'
INFO: 2020-01-12 13:34:51,967: log_options() 'quotes_favorites_file = /home/harold/.config/variety/favorite_quotes.txt'
INFO: 2020-01-12 13:34:51,967: log_options() 'quotes_font = Bitstream Charter 30'
INFO: 2020-01-12 13:34:51,967: log_options() 'quotes_hpos = 100'
INFO: 2020-01-12 13:34:51,967: log_options() 'quotes_max_length = 250'
INFO: 2020-01-12 13:34:51,967: log_options() 'quotes_tags = '
INFO: 2020-01-12 13:34:51,967: log_options() 'quotes_text_color = [255, 255, 255]'
INFO: 2020-01-12 13:34:51,967: log_options() 'quotes_text_shadow = False'
INFO: 2020-01-12 13:34:51,968: log_options() 'quotes_vpos = 40'
INFO: 2020-01-12 13:34:51,968: log_options() 'quotes_width = 70'
INFO: 2020-01-12 13:34:51,968: log_options() 'safe_mode = False'
INFO: 2020-01-12 13:34:51,968: log_options() 'slideshow_custom_enabled = False'
INFO: 2020-01-12 13:34:51,968: log_options() 'slideshow_custom_folder = /home/harold/Pictures'
INFO: 2020-01-12 13:34:51,968: log_options() 'slideshow_downloads_enabled = False'
INFO: 2020-01-12 13:34:51,968: log_options() 'slideshow_fade = 0.4'
INFO: 2020-01-12 13:34:51,968: log_options() 'slideshow_favorites_enabled = True'
INFO: 2020-01-12 13:34:51,968: log_options() 'slideshow_mode = Fullscreen'
INFO: 2020-01-12 13:34:51,968: log_options() 'slideshow_monitor = All'
INFO: 2020-01-12 13:34:51,968: log_options() 'slideshow_pan = 0.05'
INFO: 2020-01-12 13:34:51,968: log_options() 'slideshow_seconds = 6.0'
INFO: 2020-01-12 13:34:51,968: log_options() 'slideshow_sort_order = Random'
INFO: 2020-01-12 13:34:51,968: log_options() 'slideshow_sources_enabled = True'
INFO: 2020-01-12 13:34:51,968: log_options() 'slideshow_zoom = 0.2'
INFO: 2020-01-12 13:34:51,968: log_options() 'smart_enabled = False'
INFO: 2020-01-12 13:34:51,969: log_options() 'smart_notice_shown = False'
INFO: 2020-01-12 13:34:51,969: log_options() 'smart_register_shown = False'
INFO: 2020-01-12 13:34:51,969: log_options() 'sources = [[False, 'apod', "NASA's Astronomy Picture of the Day"], [False, 'bing', 'Bing Photo of the Day'], [False, 'chromeos', 'Chrome OS Wallpapers'], [False, 'desktoppr', 'Random wallpapers from Desktoppr.co'], [False, 'earth', 'World Sunlight Map - live wallpaper from Die.net'], [False, 'earthview', 'Google Earth View Wallpapers'], [False, 'unsplash', 'High-resolution photos from Unsplash.com'], [False, 'favorites', 'The Favorites folder'], [False, 'fetched', 'The Fetched folder'], [True, 'image', '/home/harold/Pictures/壁纸A/wolf_silhouette_hills_130119_1920x1080.jpg'], [True, 'image', '/home/harold/Pictures/壁纸A/wolf_silhouette_art_138559_1920x1080.jpg'], [True, 'image', '/home/harold/Pictures/壁纸A/wolf_art_muzzle_142979_1920x1080.jpg'], [True, 'image', '/home/harold/Pictures/壁纸A/firewatch-landscape-forest-minimalistic-games-6993.jpg'], [True, 'image', '/home/harold/Pictures/壁纸A/firewatch-forest-landscape-in-game-minimalistic-games-7558.jpg'], [True, 'image', '/home/harold/Pictures/壁纸A/anime-landscape-girl-mountain-relaxing-trees-clouds-scenic-27286.jpeg'], [True, 'image', '/home/harold/Pictures/壁纸A/anime-girl-crying-tears-sad-sunset-clouds-scenic-sky-31230.jpeg'], [True, 'image', '/home/harold/Pictures/壁纸A/anime.jpg'], [True, 'image', '/home/harold/Pictures/壁纸A/龙猫.png'], [True, 'image', '/home/harold/Pictures/壁纸A/枫树.png'], [True, 'image', '/home/harold/Pictures/壁纸A/1565511836874.jpg']]'
INFO: 2020-01-12 13:34:51,969: log_options() 'stats_enabled = False'
INFO: 2020-01-12 13:34:51,969: log_options() 'stats_notice_shown = False'
INFO: 2020-01-12 13:34:51,969: log_options() 'sync_enabled = False'
INFO: 2020-01-12 13:34:51,969: log_options() 'use_landscape_enabled = True'
INFO: 2020-01-12 13:34:51,969: reload_config() 'No need to clear prepared queue'
INFO: 2020-01-12 13:34:51,969: clock_thread_method() 'clock thread running'
INFO: 2020-01-12 13:34:51,970: update_indicator() 'Setting file info to: /home/harold/Pictures/壁纸A/anime-landscape-girl-mountain-relaxing-trees-clouds-scenic-27286.jpeg'
INFO: 2020-01-12 13:34:51,979: load_last_change_time() 'Change interval < 6 hours, ignore persisted last_change_time, wait initially the whole interval: 300'
INFO: 2020-01-12 13:34:51,979: update_indicator() 'Setting file info to: /home/harold/Pictures/壁纸A/anime-landscape-girl-mountain-relaxing-trees-clouds-scenic-27286.jpeg'
INFO: 2020-01-12 13:34:51,983: regular_change_thread() 'regular_change thread running'
INFO: 2020-01-12 13:34:51,985: prepare_thread() 'Prepare thread running'
INFO: 2020-01-12 13:34:51,987: prepare_thread() 'Prepared buffer contains 0 images'
INFO: 2020-01-12 13:34:51,987: prepare_thread() 'Preparing some images'
INFO: 2020-01-12 13:34:52,012: update_indicator_icon() 'Creating indicator'
INFO: 2020-01-12 13:34:52,041: prepare_thread() 'After search prepared buffer contains 11 images'
WARNING: 2020-01-12 13:34:52,084: create_menu() 'Variety Slideshow is not installed. This is an optional extension adding pan-and-zoom slideshows to Variety: see https://github.com/peterlevi/variety-slideshow for details'
INFO: 2020-01-12 13:34:52,091: create_indicator() 'indicator backend: AppIndicator3'
INFO: 2020-01-12 13:34:52,110: do_set_wp() 'Calling do_set_wp with /home/harold/Pictures/壁纸A/anime-landscape-girl-mountain-relaxing-trees-clouds-scenic-27286.jpeg, time: 1578836092.1106913'
INFO: 2020-01-12 13:34:52,129: compute_trimmed_offsets() 'Trimmed offsets debug info: w:1920, h:1080, ratio:1.777778, iw:2500, ih:1384, scw:1950, sch:1080, ho:15, vo:0'
INFO: 2020-01-12 13:34:52,130: build_imagemagick_clock_cmd() 'Applying clock filter: -density 100 -font `fc-match -f '%{file[0]}' 'Sans:Bold:70'` -pointsize 70 -gravity SouthEast -fill '#00000044' -annotate 0x0+73+108 '13:34' -fill white -annotate 0x0+75+110 '13:34' -font `fc-match -f '%{file[0]}' 'Sans:32'` -pointsize 32 -fill '#00000044' -annotate 0x0+73+58 '星期日, 一月 12' -fill white -annotate 0x0+75+60 '星期日, 一月 12''
INFO: 2020-01-12 13:34:52,130: build_imagemagick_clock_cmd() 'ImageMagick clock cmd: convert '/home/harold/Pictures/壁纸A/anime-landscape-girl-mountain-relaxing-trees-clouds-scenic-27286.jpeg' -scale 1920x1080^ -density 100 -font `fc-match -f '%{file[0]}' 'Sans:Bold:70'` -pointsize 70 -gravity SouthEast -fill '#00000044' -annotate 0x0+73+108 '13:34' -fill white -annotate 0x0+75+110 '13:34' -font `fc-match -f '%{file[0]}' 'Sans:32'` -pointsize 32 -fill '#00000044' -annotate 0x0+73+58 '星期日, 一月 12' -fill white -annotate 0x0+75+60 '星期日, 一月 12' /home/harold/.config/variety/wallpaper/wallpaper-clock-3a428e308c0f59c3e9b67eccfdff8567.jpg'
INFO: 2020-01-12 13:34:52,330: set_from_theme_icon() 'Showing indicator icon variety-indicator from GTK theme'
INFO: 2020-01-12 13:34:52,971: clock_thread_method() 'clock_thread updates wallpaper'
INFO: 2020-01-12 13:34:53,112: do_set_wp() 'Calling do_set_wp with /home/harold/Pictures/壁纸A/anime-landscape-girl-mountain-relaxing-trees-clouds-scenic-27286.jpeg, time: 1578836093.1120317'
INFO: 2020-01-12 13:34:53,500: update_indicator() 'Setting file info to: /home/harold/Pictures/壁纸A/anime-landscape-girl-mountain-relaxing-trees-clouds-scenic-27286.jpeg'
INFO: 2020-01-12 13:34:53,515: list_files() 'More than 1 files in the folders, stop listing'
INFO: 2020-01-12 13:34:54,044: trigger_download() 'Triggering download thread to check if download needed'
INFO: 2020-01-12 13:34:54,643: reload() 'Reloading preferences dialog'
INFO: 2020-01-12 13:34:56,181: compute_trimmed_offsets() 'Trimmed offsets debug info: w:1920, h:1080, ratio:1.777778, iw:2500, ih:1384, scw:1950, sch:1080, ho:15, vo:0'
INFO: 2020-01-12 13:34:56,181: build_imagemagick_clock_cmd() 'Applying clock filter: -density 100 -font `fc-match -f '%{file[0]}' 'Sans:Bold:70'` -pointsize 70 -gravity SouthEast -fill '#00000044' -annotate 0x0+73+108 '13:34' -fill white -annotate 0x0+75+110 '13:34' -font `fc-match -f '%{file[0]}' 'Sans:32'` -pointsize 32 -fill '#00000044' -annotate 0x0+73+58 '星期日, 一月 12' -fill white -annotate 0x0+75+60 '星期日, 一月 12''
INFO: 2020-01-12 13:34:56,181: build_imagemagick_clock_cmd() 'ImageMagick clock cmd: convert '/home/harold/Pictures/壁纸A/anime-landscape-girl-mountain-relaxing-trees-clouds-scenic-27286.jpeg' -scale 1920x1080^ -density 100 -font `fc-match -f '%{file[0]}' 'Sans:Bold:70'` -pointsize 70 -gravity SouthEast -fill '#00000044' -annotate 0x0+73+108 '13:34' -fill white -annotate 0x0+75+110 '13:34' -font `fc-match -f '%{file[0]}' 'Sans:32'` -pointsize 32 -fill '#00000044' -annotate 0x0+73+58 '星期日, 一月 12' -fill white -annotate 0x0+75+60 '星期日, 一月 12' /home/harold/.config/variety/wallpaper/wallpaper-clock-12ebd6d9828d165fccefc003a2b26af9.jpg'
INFO: 2020-01-12 13:34:56,619: update_indicator() 'Setting file info to: /home/harold/Pictures/壁纸A/anime-landscape-girl-mountain-relaxing-trees-clouds-scenic-27286.jpeg'
INFO: 2020-01-12 13:34:56,620: list_files() 'More than 1 files in the folders, stop listing'
INFO: 2020-01-12 13:35:00,984: clock_thread_method() 'clock_thread updates wallpaper'
INFO: 2020-01-12 13:35:00,985: do_set_wp() 'Calling do_set_wp with /home/harold/Pictures/壁纸A/anime-landscape-girl-mountain-relaxing-trees-clouds-scenic-27286.jpeg, time: 1578836100.9854243'
INFO: 2020-01-12 13:35:00,987: compute_trimmed_offsets() 'Trimmed offsets debug info: w:1920, h:1080, ratio:1.777778, iw:2500, ih:1384, scw:1950, sch:1080, ho:15, vo:0'
INFO: 2020-01-12 13:35:00,987: build_imagemagick_clock_cmd() 'Applying clock filter: -density 100 -font `fc-match -f '%{file[0]}' 'Sans:Bold:70'` -pointsize 70 -gravity SouthEast -fill '#00000044' -annotate 0x0+73+108 '13:35' -fill white -annotate 0x0+75+110 '13:35' -font `fc-match -f '%{file[0]}' 'Sans:32'` -pointsize 32 -fill '#00000044' -annotate 0x0+73+58 '星期日, 一月 12' -fill white -annotate 0x0+75+60 '星期日, 一月 12''
INFO: 2020-01-12 13:35:00,987: build_imagemagick_clock_cmd() 'ImageMagick clock cmd: convert '/home/harold/Pictures/壁纸A/anime-landscape-girl-mountain-relaxing-trees-clouds-scenic-27286.jpeg' -scale 1920x1080^ -density 100 -font `fc-match -f '%{file[0]}' 'Sans:Bold:70'` -pointsize 70 -gravity SouthEast -fill '#00000044' -annotate 0x0+73+108 '13:35' -fill white -annotate 0x0+75+110 '13:35' -font `fc-match -f '%{file[0]}' 'Sans:32'` -pointsize 32 -fill '#00000044' -annotate 0x0+73+58 '星期日, 一月 12' -fill white -annotate 0x0+75+60 '星期日, 一月 12' /home/harold/.config/variety/wallpaper/wallpaper-clock-32704bdf8c2466c04b8b91a8edcd4711.jpg'
INFO: 2020-01-12 13:35:01,364: update_indicator() 'Setting file info to: /home/harold/Pictures/壁纸A/anime-landscape-girl-mountain-relaxing-trees-clouds-scenic-27286.jpeg'
INFO: 2020-01-12 13:35:01,365: list_files() 'More than 1 files in the folders, stop listing'
INFO: 2020-01-12 13:35:12,015: server_options_thread() 'Fetching server options from http://tiny.cc/variety-options-063'
ERROR: 2020-01-12 13:35:17,106: server_options_thread() 'Could not fetch Variety serverside options'
Traceback (most recent call last):
  File "/usr/lib/python3.8/site-packages/urllib3/connection.py", line 156, in _new_conn
    conn = connection.create_connection(
  File "/usr/lib/python3.8/site-packages/urllib3/util/connection.py", line 84, in create_connection
    raise err
  File "/usr/lib/python3.8/site-packages/urllib3/util/connection.py", line 74, in create_connection
    sock.connect(sa)
socket.timeout: timed out

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/lib/python3.8/site-packages/urllib3/connectionpool.py", line 665, in urlopen
    httplib_response = self._make_request(
  File "/usr/lib/python3.8/site-packages/urllib3/connectionpool.py", line 387, in _make_request
    conn.request(method, url, **httplib_request_kw)
  File "/usr/lib/python3.8/http/client.py", line 1230, in request
    self._send_request(method, url, body, headers, encode_chunked)
  File "/usr/lib/python3.8/http/client.py", line 1276, in _send_request
    self.endheaders(body, encode_chunked=encode_chunked)
  File "/usr/lib/python3.8/http/client.py", line 1225, in endheaders
    self._send_output(message_body, encode_chunked=encode_chunked)
  File "/usr/lib/python3.8/http/client.py", line 1004, in _send_output
    self.send(msg)
  File "/usr/lib/python3.8/http/client.py", line 944, in send
    self.connect()
  File "/usr/lib/python3.8/site-packages/urllib3/connection.py", line 184, in connect
    conn = self._new_conn()
  File "/usr/lib/python3.8/site-packages/urllib3/connection.py", line 161, in _new_conn
    raise ConnectTimeoutError(
urllib3.exceptions.ConnectTimeoutError: (<urllib3.connection.HTTPConnection object at 0x7f22c0bf2040>, 'Connection to tiny.cc timed out. (connect timeout=5)')

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/lib/python3.8/site-packages/requests/adapters.py", line 439, in send
    resp = conn.urlopen(
  File "/usr/lib/python3.8/site-packages/urllib3/connectionpool.py", line 719, in urlopen
    retries = retries.increment(
  File "/usr/lib/python3.8/site-packages/urllib3/util/retry.py", line 436, in increment
    raise MaxRetryError(_pool, url, error or ResponseError(cause))
urllib3.exceptions.MaxRetryError: HTTPConnectionPool(host='tiny.cc', port=80): Max retries exceeded with url: /variety-options-063 (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f22c0bf2040>, 'Connection to tiny.cc timed out. (connect timeout=5)'))

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/lib/python3.8/site-packages/variety/VarietyWindow.py", line 1087, in server_options_thread
    self.server_options = Util.fetch_json(VarietyWindow.SERVERSIDE_OPTIONS_URL)
  File "/usr/lib/python3.8/site-packages/variety/Util.py", line 643, in fetch_json
    return Util.request(url, data, **request_kwargs).json()
  File "/usr/lib/python3.8/site-packages/variety/Util.py", line 612, in request
    r = requests.request(
  File "/usr/lib/python3.8/site-packages/requests/api.py", line 60, in request
    return session.request(method=method, url=url, **kwargs)
  File "/usr/lib/python3.8/site-packages/requests/sessions.py", line 533, in request
    resp = self.send(prep, **send_kwargs)
  File "/usr/lib/python3.8/site-packages/requests/sessions.py", line 646, in send
    r = adapter.send(request, **kwargs)
  File "/usr/lib/python3.8/site-packages/requests/adapters.py", line 504, in send
    raise ConnectTimeout(e, request=request)
requests.exceptions.ConnectTimeout: HTTPConnectionPool(host='tiny.cc', port=80): Max retries exceeded with url: /variety-options-063 (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f22c0bf2040>, 'Connection to tiny.cc timed out. (connect timeout=5)'))
INFO: 2020-01-12 13:35:47,139: server_options_thread() 'Fetching server options from http://tiny.cc/variety-options-063'
ERROR: 2020-01-12 13:35:52,157: server_options_thread() 'Could not fetch Variety serverside options'
Traceback (most recent call last):
  File "/usr/lib/python3.8/site-packages/urllib3/connection.py", line 156, in _new_conn
    conn = connection.create_connection(
  File "/usr/lib/python3.8/site-packages/urllib3/util/connection.py", line 84, in create_connection
    raise err
  File "/usr/lib/python3.8/site-packages/urllib3/util/connection.py", line 74, in create_connection
    sock.connect(sa)
OSError: [Errno 101] 网络不可达

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/lib/python3.8/site-packages/urllib3/connectionpool.py", line 665, in urlopen
    httplib_response = self._make_request(
  File "/usr/lib/python3.8/site-packages/urllib3/connectionpool.py", line 387, in _make_request
    conn.request(method, url, **httplib_request_kw)
  File "/usr/lib/python3.8/http/client.py", line 1230, in request
    self._send_request(method, url, body, headers, encode_chunked)
  File "/usr/lib/python3.8/http/client.py", line 1276, in _send_request
    self.endheaders(body, encode_chunked=encode_chunked)
  File "/usr/lib/python3.8/http/client.py", line 1225, in endheaders
    self._send_output(message_body, encode_chunked=encode_chunked)
  File "/usr/lib/python3.8/http/client.py", line 1004, in _send_output
    self.send(msg)
  File "/usr/lib/python3.8/http/client.py", line 944, in send
    self.connect()
  File "/usr/lib/python3.8/site-packages/urllib3/connection.py", line 184, in connect
    conn = self._new_conn()
  File "/usr/lib/python3.8/site-packages/urllib3/connection.py", line 168, in _new_conn
    raise NewConnectionError(
urllib3.exceptions.NewConnectionError: <urllib3.connection.HTTPConnection object at 0x7f22c0bf2c70>: Failed to establish a new connection: [Errno 101] 网络不可达

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/lib/python3.8/site-packages/requests/adapters.py", line 439, in send
    resp = conn.urlopen(
  File "/usr/lib/python3.8/site-packages/urllib3/connectionpool.py", line 719, in urlopen
    retries = retries.increment(
  File "/usr/lib/python3.8/site-packages/urllib3/util/retry.py", line 436, in increment
    raise MaxRetryError(_pool, url, error or ResponseError(cause))
urllib3.exceptions.MaxRetryError: HTTPConnectionPool(host='tiny.cc', port=80): Max retries exceeded with url: /variety-options-063 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f22c0bf2c70>: Failed to establish a new connection: [Errno 101] 网络不可达'))

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/lib/python3.8/site-packages/variety/VarietyWindow.py", line 1087, in server_options_thread
    self.server_options = Util.fetch_json(VarietyWindow.SERVERSIDE_OPTIONS_URL)
  File "/usr/lib/python3.8/site-packages/variety/Util.py", line 643, in fetch_json
    return Util.request(url, data, **request_kwargs).json()
  File "/usr/lib/python3.8/site-packages/variety/Util.py", line 612, in request
    r = requests.request(
  File "/usr/lib/python3.8/site-packages/requests/api.py", line 60, in request
    return session.request(method=method, url=url, **kwargs)
  File "/usr/lib/python3.8/site-packages/requests/sessions.py", line 533, in request
    resp = self.send(prep, **send_kwargs)
  File "/usr/lib/python3.8/site-packages/requests/sessions.py", line 646, in send
    r = adapter.send(request, **kwargs)
  File "/usr/lib/python3.8/site-packages/requests/adapters.py", line 516, in send
    raise ConnectionError(e, request=request)
requests.exceptions.ConnectionError: HTTPConnectionPool(host='tiny.cc', port=80): Max retries exceeded with url: /variety-options-063 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f22c0bf2c70>: Failed to establish a new connection: [Errno 101] 网络不可达'))
INFO: 2020-01-12 13:36:00,048: clock_thread_method() 'clock_thread updates wallpaper'
INFO: 2020-01-12 13:36:00,049: do_set_wp() 'Calling do_set_wp with /home/harold/Pictures/壁纸A/anime-landscape-girl-mountain-relaxing-trees-clouds-scenic-27286.jpeg, time: 1578836160.0493681'
INFO: 2020-01-12 13:36:00,053: compute_trimmed_offsets() 'Trimmed offsets debug info: w:1920, h:1080, ratio:1.777778, iw:2500, ih:1384, scw:1950, sch:1080, ho:15, vo:0'
INFO: 2020-01-12 13:36:00,053: build_imagemagick_clock_cmd() 'Applying clock filter: -density 100 -font `fc-match -f '%{file[0]}' 'Sans:Bold:70'` -pointsize 70 -gravity SouthEast -fill '#00000044' -annotate 0x0+73+108 '13:36' -fill white -annotate 0x0+75+110 '13:36' -font `fc-match -f '%{file[0]}' 'Sans:32'` -pointsize 32 -fill '#00000044' -annotate 0x0+73+58 '星期日, 一月 12' -fill white -annotate 0x0+75+60 '星期日, 一月 12''
INFO: 2020-01-12 13:36:00,054: build_imagemagick_clock_cmd() 'ImageMagick clock cmd: convert '/home/harold/Pictures/壁纸A/anime-landscape-girl-mountain-relaxing-trees-clouds-scenic-27286.jpeg' -scale 1920x1080^ -density 100 -font `fc-match -f '%{file[0]}' 'Sans:Bold:70'` -pointsize 70 -gravity SouthEast -fill '#00000044' -annotate 0x0+73+108 '13:36' -fill white -annotate 0x0+75+110 '13:36' -font `fc-match -f '%{file[0]}' 'Sans:32'` -pointsize 32 -fill '#00000044' -annotate 0x0+73+58 '星期日, 一月 12' -fill white -annotate 0x0+75+60 '星期日, 一月 12' /home/harold/.config/variety/wallpaper/wallpaper-clock-e226e9a8446c970fe89cbea25b1a79ea.jpg'
INFO: 2020-01-12 13:36:00,350: update_indicator() 'Setting file info to: /home/harold/Pictures/壁纸A/anime-landscape-girl-mountain-relaxing-trees-clouds-scenic-27286.jpeg'
INFO: 2020-01-12 13:36:00,351: list_files() 'More than 1 files in the folders, stop listing'
INFO: 2020-01-12 13:36:22,189: server_options_thread() 'Fetching server options from http://tiny.cc/variety-options-063'
ERROR: 2020-01-12 13:36:27,206: server_options_thread() 'Could not fetch Variety serverside options'
Traceback (most recent call last):
  File "/usr/lib/python3.8/site-packages/urllib3/connection.py", line 156, in _new_conn
    conn = connection.create_connection(
  File "/usr/lib/python3.8/site-packages/urllib3/util/connection.py", line 84, in create_connection
    raise err
  File "/usr/lib/python3.8/site-packages/urllib3/util/connection.py", line 74, in create_connection
    sock.connect(sa)
OSError: [Errno 101] 网络不可达

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/lib/python3.8/site-packages/urllib3/connectionpool.py", line 665, in urlopen
    httplib_response = self._make_request(
  File "/usr/lib/python3.8/site-packages/urllib3/connectionpool.py", line 387, in _make_request
    conn.request(method, url, **httplib_request_kw)
  File "/usr/lib/python3.8/http/client.py", line 1230, in request
    self._send_request(method, url, body, headers, encode_chunked)
  File "/usr/lib/python3.8/http/client.py", line 1276, in _send_request
    self.endheaders(body, encode_chunked=encode_chunked)
  File "/usr/lib/python3.8/http/client.py", line 1225, in endheaders
    self._send_output(message_body, encode_chunked=encode_chunked)
  File "/usr/lib/python3.8/http/client.py", line 1004, in _send_output
    self.send(msg)
  File "/usr/lib/python3.8/http/client.py", line 944, in send
    self.connect()
  File "/usr/lib/python3.8/site-packages/urllib3/connection.py", line 184, in connect
    conn = self._new_conn()
  File "/usr/lib/python3.8/site-packages/urllib3/connection.py", line 168, in _new_conn
    raise NewConnectionError(
urllib3.exceptions.NewConnectionError: <urllib3.connection.HTTPConnection object at 0x7f22c01d3490>: Failed to establish a new connection: [Errno 101] 网络不可达

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/lib/python3.8/site-packages/requests/adapters.py", line 439, in send
    resp = conn.urlopen(
  File "/usr/lib/python3.8/site-packages/urllib3/connectionpool.py", line 719, in urlopen
    retries = retries.increment(
  File "/usr/lib/python3.8/site-packages/urllib3/util/retry.py", line 436, in increment
    raise MaxRetryError(_pool, url, error or ResponseError(cause))
urllib3.exceptions.MaxRetryError: HTTPConnectionPool(host='tiny.cc', port=80): Max retries exceeded with url: /variety-options-063 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f22c01d3490>: Failed to establish a new connection: [Errno 101] 网络不可达'))

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/lib/python3.8/site-packages/variety/VarietyWindow.py", line 1087, in server_options_thread
    self.server_options = Util.fetch_json(VarietyWindow.SERVERSIDE_OPTIONS_URL)
  File "/usr/lib/python3.8/site-packages/variety/Util.py", line 643, in fetch_json
    return Util.request(url, data, **request_kwargs).json()
  File "/usr/lib/python3.8/site-packages/variety/Util.py", line 612, in request
    r = requests.request(
  File "/usr/lib/python3.8/site-packages/requests/api.py", line 60, in request
    return session.request(method=method, url=url, **kwargs)
  File "/usr/lib/python3.8/site-packages/requests/sessions.py", line 533, in request
    resp = self.send(prep, **send_kwargs)
  File "/usr/lib/python3.8/site-packages/requests/sessions.py", line 646, in send
    r = adapter.send(request, **kwargs)
  File "/usr/lib/python3.8/site-packages/requests/adapters.py", line 516, in send
    raise ConnectionError(e, request=request)
requests.exceptions.ConnectionError: HTTPConnectionPool(host='tiny.cc', port=80): Max retries exceeded with url: /variety-options-063 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f22c01d3490>: Failed to establish a new connection: [Errno 101] 网络不可达'))
INFO: 2020-01-12 13:36:57,239: server_options_thread() 'Fetching server options from http://tiny.cc/variety-options-063'
INFO: 2020-01-12 13:37:00,116: clock_thread_method() 'clock_thread updates wallpaper'
INFO: 2020-01-12 13:37:00,116: do_set_wp() 'Calling do_set_wp with /home/harold/Pictures/壁纸A/anime-landscape-girl-mountain-relaxing-trees-clouds-scenic-27286.jpeg, time: 1578836220.116917'
INFO: 2020-01-12 13:37:00,118: compute_trimmed_offsets() 'Trimmed offsets debug info: w:1920, h:1080, ratio:1.777778, iw:2500, ih:1384, scw:1950, sch:1080, ho:15, vo:0'
INFO: 2020-01-12 13:37:00,119: build_imagemagick_clock_cmd() 'Applying clock filter: -density 100 -font `fc-match -f '%{file[0]}' 'Sans:Bold:70'` -pointsize 70 -gravity SouthEast -fill '#00000044' -annotate 0x0+73+108 '13:37' -fill white -annotate 0x0+75+110 '13:37' -font `fc-match -f '%{file[0]}' 'Sans:32'` -pointsize 32 -fill '#00000044' -annotate 0x0+73+58 '星期日, 一月 12' -fill white -annotate 0x0+75+60 '星期日, 一月 12''
INFO: 2020-01-12 13:37:00,119: build_imagemagick_clock_cmd() 'ImageMagick clock cmd: convert '/home/harold/Pictures/壁纸A/anime-landscape-girl-mountain-relaxing-trees-clouds-scenic-27286.jpeg' -scale 1920x1080^ -density 100 -font `fc-match -f '%{file[0]}' 'Sans:Bold:70'` -pointsize 70 -gravity SouthEast -fill '#00000044' -annotate 0x0+73+108 '13:37' -fill white -annotate 0x0+75+110 '13:37' -font `fc-match -f '%{file[0]}' 'Sans:32'` -pointsize 32 -fill '#00000044' -annotate 0x0+73+58 '星期日, 一月 12' -fill white -annotate 0x0+75+60 '星期日, 一月 12' /home/harold/.config/variety/wallpaper/wallpaper-clock-0e5fdf15421a9f3d9659d3055f6044a8.jpg'
INFO: 2020-01-12 13:37:00,471: update_indicator() 'Setting file info to: /home/harold/Pictures/壁纸A/anime-landscape-girl-mountain-relaxing-trees-clouds-scenic-27286.jpeg'
INFO: 2020-01-12 13:37:00,472: list_files() 'More than 1 files in the folders, stop listing'
ERROR: 2020-01-12 13:37:04,351: server_options_thread() 'Could not fetch Variety serverside options'
Traceback (most recent call last):
  File "/usr/lib/python3.8/site-packages/urllib3/connection.py", line 156, in _new_conn
    conn = connection.create_connection(
  File "/usr/lib/python3.8/site-packages/urllib3/util/connection.py", line 84, in create_connection
    raise err
  File "/usr/lib/python3.8/site-packages/urllib3/util/connection.py", line 74, in create_connection
    sock.connect(sa)
OSError: [Errno 101] 网络不可达

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/lib/python3.8/site-packages/urllib3/connectionpool.py", line 665, in urlopen
    httplib_response = self._make_request(
  File "/usr/lib/python3.8/site-packages/urllib3/connectionpool.py", line 387, in _make_request
    conn.request(method, url, **httplib_request_kw)
  File "/usr/lib/python3.8/http/client.py", line 1230, in request
    self._send_request(method, url, body, headers, encode_chunked)
  File "/usr/lib/python3.8/http/client.py", line 1276, in _send_request
    self.endheaders(body, encode_chunked=encode_chunked)
  File "/usr/lib/python3.8/http/client.py", line 1225, in endheaders
    self._send_output(message_body, encode_chunked=encode_chunked)
  File "/usr/lib/python3.8/http/client.py", line 1004, in _send_output
    self.send(msg)
  File "/usr/lib/python3.8/http/client.py", line 944, in send
    self.connect()
  File "/usr/lib/python3.8/site-packages/urllib3/connection.py", line 184, in connect
    conn = self._new_conn()
  File "/usr/lib/python3.8/site-packages/urllib3/connection.py", line 168, in _new_conn
    raise NewConnectionError(
urllib3.exceptions.NewConnectionError: <urllib3.connection.HTTPConnection object at 0x7f22c01d3c10>: Failed to establish a new connection: [Errno 101] 网络不可达

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/lib/python3.8/site-packages/requests/adapters.py", line 439, in send
    resp = conn.urlopen(
  File "/usr/lib/python3.8/site-packages/urllib3/connectionpool.py", line 719, in urlopen
    retries = retries.increment(
  File "/usr/lib/python3.8/site-packages/urllib3/util/retry.py", line 436, in increment
    raise MaxRetryError(_pool, url, error or ResponseError(cause))
urllib3.exceptions.MaxRetryError: HTTPConnectionPool(host='tiny.cc', port=80): Max retries exceeded with url: /variety-options-063 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f22c01d3c10>: Failed to establish a new connection: [Errno 101] 网络不可达'))

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/lib/python3.8/site-packages/variety/VarietyWindow.py", line 1087, in server_options_thread
    self.server_options = Util.fetch_json(VarietyWindow.SERVERSIDE_OPTIONS_URL)
  File "/usr/lib/python3.8/site-packages/variety/Util.py", line 643, in fetch_json
    return Util.request(url, data, **request_kwargs).json()
  File "/usr/lib/python3.8/site-packages/variety/Util.py", line 612, in request
    r = requests.request(
  File "/usr/lib/python3.8/site-packages/requests/api.py", line 60, in request
    return session.request(method=method, url=url, **kwargs)
  File "/usr/lib/python3.8/site-packages/requests/sessions.py", line 533, in request
    resp = self.send(prep, **send_kwargs)
  File "/usr/lib/python3.8/site-packages/requests/sessions.py", line 646, in send
    r = adapter.send(request, **kwargs)
  File "/usr/lib/python3.8/site-packages/requests/adapters.py", line 516, in send
    raise ConnectionError(e, request=request)
requests.exceptions.ConnectionError: HTTPConnectionPool(host='tiny.cc', port=80): Max retries exceeded with url: /variety-options-063 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f22c01d3c10>: Failed to establish a new connection: [Errno 101] 网络不可达'))
INFO: 2020-01-12 13:37:34,384: server_options_thread() 'Fetching server options from http://tiny.cc/variety-options-063'
ERROR: 2020-01-12 13:37:39,400: server_options_thread() 'Could not fetch Variety serverside options'
Traceback (most recent call last):
  File "/usr/lib/python3.8/site-packages/urllib3/connection.py", line 156, in _new_conn
    conn = connection.create_connection(
  File "/usr/lib/python3.8/site-packages/urllib3/util/connection.py", line 84, in create_connection
    raise err
  File "/usr/lib/python3.8/site-packages/urllib3/util/connection.py", line 74, in create_connection
    sock.connect(sa)
OSError: [Errno 101] 网络不可达

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/lib/python3.8/site-packages/urllib3/connectionpool.py", line 665, in urlopen
    httplib_response = self._make_request(
  File "/usr/lib/python3.8/site-packages/urllib3/connectionpool.py", line 387, in _make_request
    conn.request(method, url, **httplib_request_kw)
  File "/usr/lib/python3.8/http/client.py", line 1230, in request
    self._send_request(method, url, body, headers, encode_chunked)
  File "/usr/lib/python3.8/http/client.py", line 1276, in _send_request
    self.endheaders(body, encode_chunked=encode_chunked)
  File "/usr/lib/python3.8/http/client.py", line 1225, in endheaders
    self._send_output(message_body, encode_chunked=encode_chunked)
  File "/usr/lib/python3.8/http/client.py", line 1004, in _send_output
    self.send(msg)
  File "/usr/lib/python3.8/http/client.py", line 944, in send
    self.connect()
  File "/usr/lib/python3.8/site-packages/urllib3/connection.py", line 184, in connect
    conn = self._new_conn()
  File "/usr/lib/python3.8/site-packages/urllib3/connection.py", line 168, in _new_conn
    raise NewConnectionError(
urllib3.exceptions.NewConnectionError: <urllib3.connection.HTTPConnection object at 0x7f22c01d71c0>: Failed to establish a new connection: [Errno 101] 网络不可达

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/lib/python3.8/site-packages/requests/adapters.py", line 439, in send
    resp = conn.urlopen(
  File "/usr/lib/python3.8/site-packages/urllib3/connectionpool.py", line 719, in urlopen
    retries = retries.increment(
  File "/usr/lib/python3.8/site-packages/urllib3/util/retry.py", line 436, in increment
    raise MaxRetryError(_pool, url, error or ResponseError(cause))
urllib3.exceptions.MaxRetryError: HTTPConnectionPool(host='tiny.cc', port=80): Max retries exceeded with url: /variety-options-063 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f22c01d71c0>: Failed to establish a new connection: [Errno 101] 网络不可达'))

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/lib/python3.8/site-packages/variety/VarietyWindow.py", line 1087, in server_options_thread
    self.server_options = Util.fetch_json(VarietyWindow.SERVERSIDE_OPTIONS_URL)
  File "/usr/lib/python3.8/site-packages/variety/Util.py", line 643, in fetch_json
    return Util.request(url, data, **request_kwargs).json()
  File "/usr/lib/python3.8/site-packages/variety/Util.py", line 612, in request
    r = requests.request(
  File "/usr/lib/python3.8/site-packages/requests/api.py", line 60, in request
    return session.request(method=method, url=url, **kwargs)
  File "/usr/lib/python3.8/site-packages/requests/sessions.py", line 533, in request
    resp = self.send(prep, **send_kwargs)
  File "/usr/lib/python3.8/site-packages/requests/sessions.py", line 646, in send
    r = adapter.send(request, **kwargs)
  File "/usr/lib/python3.8/site-packages/requests/adapters.py", line 516, in send
    raise ConnectionError(e, request=request)
requests.exceptions.ConnectionError: HTTPConnectionPool(host='tiny.cc', port=80): Max retries exceeded with url: /variety-options-063 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f22c01d71c0>: Failed to establish a new connection: [Errno 101] 网络不可达'))
INFO: 2020-01-12 13:38:00,185: clock_thread_method() 'clock_thread updates wallpaper'
INFO: 2020-01-12 13:38:00,185: do_set_wp() 'Calling do_set_wp with /home/harold/Pictures/壁纸A/anime-landscape-girl-mountain-relaxing-trees-clouds-scenic-27286.jpeg, time: 1578836280.1859522'
INFO: 2020-01-12 13:38:00,190: compute_trimmed_offsets() 'Trimmed offsets debug info: w:1920, h:1080, ratio:1.777778, iw:2500, ih:1384, scw:1950, sch:1080, ho:15, vo:0'
INFO: 2020-01-12 13:38:00,190: build_imagemagick_clock_cmd() 'Applying clock filter: -density 100 -font `fc-match -f '%{file[0]}' 'Sans:Bold:70'` -pointsize 70 -gravity SouthEast -fill '#00000044' -annotate 0x0+73+108 '13:38' -fill white -annotate 0x0+75+110 '13:38' -font `fc-match -f '%{file[0]}' 'Sans:32'` -pointsize 32 -fill '#00000044' -annotate 0x0+73+58 '星期日, 一月 12' -fill white -annotate 0x0+75+60 '星期日, 一月 12''
INFO: 2020-01-12 13:38:00,190: build_imagemagick_clock_cmd() 'ImageMagick clock cmd: convert '/home/harold/Pictures/壁纸A/anime-landscape-girl-mountain-relaxing-trees-clouds-scenic-27286.jpeg' -scale 1920x1080^ -density 100 -font `fc-match -f '%{file[0]}' 'Sans:Bold:70'` -pointsize 70 -gravity SouthEast -fill '#00000044' -annotate 0x0+73+108 '13:38' -fill white -annotate 0x0+75+110 '13:38' -font `fc-match -f '%{file[0]}' 'Sans:32'` -pointsize 32 -fill '#00000044' -annotate 0x0+73+58 '星期日, 一月 12' -fill white -annotate 0x0+75+60 '星期日, 一月 12' /home/harold/.config/variety/wallpaper/wallpaper-clock-010065c22666435c26a85fdca414a0a1.jpg'
INFO: 2020-01-12 13:38:00,484: update_indicator() 'Setting file info to: /home/harold/Pictures/壁纸A/anime-landscape-girl-mountain-relaxing-trees-clouds-scenic-27286.jpeg'
INFO: 2020-01-12 13:38:00,485: list_files() 'More than 1 files in the folders, stop listing'
INFO: 2020-01-12 13:39:00,252: clock_thread_method() 'clock_thread updates wallpaper'
INFO: 2020-01-12 13:39:00,253: do_set_wp() 'Calling do_set_wp with /home/harold/Pictures/壁纸A/anime-landscape-girl-mountain-relaxing-trees-clouds-scenic-27286.jpeg, time: 1578836340.253201'
INFO: 2020-01-12 13:39:00,255: compute_trimmed_offsets() 'Trimmed offsets debug info: w:1920, h:1080, ratio:1.777778, iw:2500, ih:1384, scw:1950, sch:1080, ho:15, vo:0'
INFO: 2020-01-12 13:39:00,255: build_imagemagick_clock_cmd() 'Applying clock filter: -density 100 -font `fc-match -f '%{file[0]}' 'Sans:Bold:70'` -pointsize 70 -gravity SouthEast -fill '#00000044' -annotate 0x0+73+108 '13:39' -fill white -annotate 0x0+75+110 '13:39' -font `fc-match -f '%{file[0]}' 'Sans:32'` -pointsize 32 -fill '#00000044' -annotate 0x0+73+58 '星期日, 一月 12' -fill white -annotate 0x0+75+60 '星期日, 一月 12''
INFO: 2020-01-12 13:39:00,255: build_imagemagick_clock_cmd() 'ImageMagick clock cmd: convert '/home/harold/Pictures/壁纸A/anime-landscape-girl-mountain-relaxing-trees-clouds-scenic-27286.jpeg' -scale 1920x1080^ -density 100 -font `fc-match -f '%{file[0]}' 'Sans:Bold:70'` -pointsize 70 -gravity SouthEast -fill '#00000044' -annotate 0x0+73+108 '13:39' -fill white -annotate 0x0+75+110 '13:39' -font `fc-match -f '%{file[0]}' 'Sans:32'` -pointsize 32 -fill '#00000044' -annotate 0x0+73+58 '星期日, 一月 12' -fill white -annotate 0x0+75+60 '星期日, 一月 12' /home/harold/.config/variety/wallpaper/wallpaper-clock-daa832ae7b2cb94e36f16f6b747f41e5.jpg'
INFO: 2020-01-12 13:39:00,576: update_indicator() 'Setting file info to: /home/harold/Pictures/壁纸A/anime-landscape-girl-mountain-relaxing-trees-clouds-scenic-27286.jpeg'
INFO: 2020-01-12 13:39:00,577: list_files() 'More than 1 files in the folders, stop listing'
